components/lcd_touch/esp_lcd_touch_gt911/esp_lcd_touch_gt911.c

```diff
@@ -132,7 +180,17 @@ static esp_err_t esp_lcd_touch_gt911_read_data(esp_lcd_touch_handle_t tp)
 
     /* Any touch data? */
     if ((buf[0] & 0x80) == 0x00) {
+#if (CONFIG_ESP_LCD_TOUCH_MAX_BUTTONS > 0)
+        tp->data.button[0].status = 0x00;
+#endif
         touch_gt911_i2c_write(tp, ESP_LCD_TOUCH_GT911_READ_XY_REG, clear);
+    } else if ((buf[0] & 0x10) == 0x10) {
```

Comment:
I am sorry, I overlooked something maybe. What does it mean 0x10 ? I checked the GT911 registers and I see this bit is reserved in 0x814E register.
I see this:
![image](https://github.com/espressif/esp-bsp/assets/98878239/f8cf7f6d-462f-4610-94c1-ad4059749518)

Reply:
> I am sorry, I overlooked something maybe. What does it mean 0x10 ? I checked the GT911 registers and I see this bit is reserved in 0x814E register. I see this: ![image](https://user-images.githubusercontent.com/98878239/269532545-f8cf7f6d-462f-4610-94c1-ad4059749518.png)

Yes, you are right. some documents do not explain this bit, it's not very clear about this key event.
![image](https://github.com/espressif/esp-bsp/assets/111102666/e8ceaf22-85a3-48ef-9588-6c3f2f962a5a)
![image](https://github.com/espressif/esp-bsp/assets/111102666/1962a8f8-a988-4a0b-852e-bcc755ed44f4)

Reply:
I cannot find newer datasheet, but it looks like only information that some key was changed. I have some older version with lot of chinesse letters, but I see, that there are supported four buttons (registers 0x8093 - 0x8096). If the GT11 controller support all four buttons, we should support all of them in this driver.

Reply:
> I cannot find newer datasheet, but it looks like only information that some key was changed. I have some older version with lot of chinesse letters, but I see, that there are supported four buttons (registers 0x8093 - 0x8096). If the GT11 controller support all four buttons, we should support all of them in this driver.

OK, I'll check this four button event.